packages/api/src/constants.js

```diff
@@ -6,8 +6,8 @@ module.exports = {
   EPOCH_CHANGE_TIME: Number(process.env.EPOCH_CHANGE_TIME),
   get genesisTime () {
     if (!genesisTime || isNaN(genesisTime)) {
-      return TenderdashRPC.getGenesis().then((genesis) => {
-        genesisTime = new Date(genesis.genesis_time)
+      return TenderdashRPC.getBlockByHeight(1).then((blockInfo) => {
+        genesisTime = new Date(blockInfo ? blockInfo.block.header.time : '')
```

Comment:
I guess it cannot be `''`, better to throw an error, because then genesis time will be shown as jan1 1970

Reply:
✅